SpeedrunPracticePlugin.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace SpeedrunPractice
 {
-  [BepInPlugin("com.aldelaro5.BugFables.plugins.SpeedrunPractice", "Speedrun Practice", "2.0.1")]
+  [BepInPlugin("com.aldelaro5.BugFables.plugins.SpeedrunPractice", "Speedrun Practice", "2.0.2")]
```

Comment:
Don't version bump as part of the commit, I will be the one to do it when I do a release to tag it.